lib/src/proxy-authenticate.class.ts

```diff
@@ -0,0 +1,48 @@
+import { BaseHeader } from "./base-header.class";
+export class ProxyAuthenticate extends BaseHeader {
+  protected _type?: string | undefined;
+  protected _realm?: string | undefined;
```

Comment:
```suggestion
  /** 
    * @description A description of the protected area, the realm. If no realm is 
    * specified, clients often display a formatted host name instead.
    * protected _realm?: string | undefined;
    */
```

Reply:
Should I use description for every headers?